common/src/main/scala/org/apache/celeborn/common/meta/WorkerInfo.scala

```diff
@@ -239,10 +239,7 @@ class WorkerInfo(
   def updateThenGetUserResourceConsumption(resourceConsumptions: util.Map[
     UserIdentifier,
     ResourceConsumption]): util.Map[UserIdentifier, ResourceConsumption] = {
-    userResourceConsumption.keys().asScala.filterNot(resourceConsumptions.containsKey).foreach {
-      identifier =>
-        userResourceConsumption.put(identifier, ResourceConsumption(0, 0, 0, 0))
-    }
+    userResourceConsumption.clear()
```

Comment:
if we always reserve the ResourceConsumption with 0, seems it would not be cleaned.